src/view/ChewingEditor.cpp

```diff
@@ -217,7 +217,6 @@ void ChewingEditor::setupUndo()
         model_, SIGNAL(undoCompleted(size_t)),
         ui_.get()->notification, SLOT(notifyUndoCompleted(size_t))
     );
-
```

Comment:
"Is it relevant to the purpose of this pull request?"
No, should I merge it to other commit?

"Use git rebase -i to rework the patches."
OK, I will do it.

Thank you!!